common/gutenberg-web-single-block/insert-block.js

```diff
@@ -0,0 +1,16 @@
+window.setTimeout( () => {
+	const blockHTML = `%@`;
```

Comment:
Interesting idea!
How would the `window.blockToEdit = ...` script look?

We still need to interpolate the block content somehow.

> to avoid using %@ (which might be incompatible with Android)

True that. I thought about using a placeholder like `\`BLOCK_HTML\`` to search and replace. Android could search and replace `%@` too. We can check with @marecar3 what's best.

Reply:
> How would the window.blockToEdit = ... script look?

Reply:
@etoledom I'm marking this conversation as resolved because I see the Android implementation is working file with the `%@` (it's replacing it with a `%s`) here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2063/files#diff-923f1fe097c6b19b90816d71b7109fe0R214